starter-code/src/merge-sort.js

```diff
@@ -1,15 +1,14 @@
 function mergeSort(array) {
   if(array.length < 2) { return array }
 
-  var middle  = Math.floor(array.length / 2);
-  var left    = array.slice(0, middle);
-  var right   = array.slice(middle);
-
+  let middle  = Math.floor(array.length / 2);
```

Comment:
`middle` can be a constant variable because you're never reassigning it anywhere.